packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -47,16 +45,19 @@ export const MenuUI = styled.div`
 
 export const MenuBackButtonUI = styled( Button )`
 	&.is-tertiary {
-		color: ${ G2.lightGray.ui };
+		color: inherit;
+		opacity: 0.85;
```

Comment:
It appears the trick here is to use opacities to define the grays, so that colors can be inherited from context instead of explicitly set to light or dark. I suspect that can work as a holdover, but it inherently becomes a bit more fuzzy, so I'd love for us to be able to feed it some specific colors.

Reply:
Yes, that's the solution. I don't think we need to be more clever than that for the moment. Adding explicit colors (and modes dark/light) is going to be harder and that component is supposed to go away anyway.

Reply:
If we can go back and tune the colors once we're happy with the overall component, it seems fine to go with the opacity solution in the mean time 👍 👍